deepchem/models/torch_models/mxmnet.py

```diff
@@ -0,0 +1,324 @@
+import math
+import deepchem as dc
+import torch
+import torch.nn as nn
+from torch_geometric.data import Batch
+from torch_geometric.nn import global_add_pool, radius
+from torch_geometric.utils import remove_self_loops
+from typing import Union, Callable, Tuple, Optional
+from deepchem.utils.pytorch_utils import get_activation
+from deepchem.models.torch_models.layers import MXMNetBesselBasisLayer, MXMNetSphericalBasisLayer, MultilayerPerceptron
+
+
+class MXMNet(nn.Module):
+    """
+    Molecular Mechanics-Driven Graph Neural Network with Multiplex Graph for Molecular Structures.
+    In this class, we define the global and local message passing layers and the MXMNet Model[1]_.
+    We also define the forward call of this model in the forward function.
+
+    Note:
+    This model currently supports only Single Task Regression. Multi Task Regression, Single-Task Classification
+    and Multi-Task Classification is not yet implemented.
+    References
+    ----------
+    .. [1] Molecular Mechanics-Driven Graph Neural Network with Multiplex Graph for Molecular Structures https://arxiv.org/pdf/2011.07457
+
+    Example
+    -------
+    >>> import deepchem as dc
+    >>> import os
+    >>> import tempfile
+    >>> import torch
+    >>> from torch_geometric.data import Data, Batch
+    >>> from deepchem.feat.molecule_featurizers import MXMNetFeaturizer
+    >>> QM9_TASKS = ["mu", "alpha", "homo", "lumo", "gap", "r2", "zpve", "cv", "u0", "u298",
+    ...              "h298", "g298"]
+    >>> device = 'cpu'
```

Comment:
Why are we setting device to cpu in the usage example?

Reply:
This was just for testing, I'll remove it. It won't cause any change.